doc/UsersGuide/source/ContributingDevelopment.rst

```diff
@@ -4,4 +4,218 @@
+
+3. When PRs are created, the creator must temporarily modify .gitmodules to point to his/her fork and branch if updates are required for submodules.
+
+4. Merging code from PRs with submodules requires coordination with the person making the PRs. From the "innermost" nested PR up to the top-level PR, the PRs need to be merged as-is. After each merge, the person creating the PRs has to update his/her local code to check out the merged version, revert the change to .gitmodules, and push this to GitHub to update the PR. And so on and so forth.
+
+5. Checking out the code ufs_release_1.0 should always be as follows:
+
+.. code-block:: console
+
+   git clone https://github.com/ufs-community/ufs-weather-model
+   cd ufs-weather-model
+   git checkout ufs_release_1.00
+   git submodule update --init --recursive
+
+6. Checking out a PR with id ID for testing it should always be as follows:
+
+.. code-block:: console
+
+   git clone https://github.com/ufs-community/ufs-weather-model
+   cd ufs-weather-model
+   git fetch origin pull/ID/head:BRANCHNAME
+   git checkout BRANCHNAME
+   git submodule update --init --recursive
+
+It is suggested that the developers inform all the related code managers as the hierarchy structure of the ufs-weather-model repository may require collaboration among the code managers.
+
+-----------------------------------
+Engaging in the code review process
+-----------------------------------
+
+When code managers receive a pull request to commit the code changes, it is recommended that they add at least two code reviewers to review the code and at least one of the reviewers has write permission. The reviewers will write comments about the code changes and give a recommendation as to whether the code changes can be committed. What kinds of code changes will be accepted in the repository is beyond the scope of this document; future ufs-weather-model code management documents may have a detailed answer for that.
+
+Reviewers may suggest some code changes during the review process. Developers need to respond to these comments in order to get code changes committed. If developers make further changes to their branch, reviewers need to check the code changes again. When both reviewers give recommendation to commit the code, code managers will merge the changes into the repository.
+
+----------------------------
+Conducting regression tests
+----------------------------
+
+Only developers using Tier 1 and 2 platforms can run the ufs-weather-model regression tests. Other developers need to work with the code managers to assure completion of the regression tests.
+
+
+To run regression test using rt.sh
+
+rt.sh is a bash shell file to run the RT and has the following options:
+
+.. code-block:: console
+
+   Usage: $0 -c <model> | -f | -s | -l <file> | -m | -r | -e | -h
```

Comment:
Please replace ``$0`` with ``./rt.sh``

Reply:
Also, ``-c`` option does not need ``<model>`` anymore. We might need to update usage message in the script itself.

Reply:
See below.